ArduCopter/mode.h

```diff
@@ -439,157 +439,44 @@ class ModeAuto : public Mode {
 };
 
 #if AUTOTUNE_ENABLED == ENABLED
+/*
+  wrapper class for AC_AutoTune
+ */
+class AutoTune : public AC_AutoTune
+{
+public:
+    bool init() override;
+    void run() override;
+
+protected:
+    bool start(void) override;
+    bool position_ok() override;
+    float get_pilot_desired_climb_rate_cms(void) const override;
+    void get_pilot_desired_rp_yrate_cd(int32_t &roll_cd, int32_t &pitch_cd, int32_t &yaw_rate_cds) override;
```

Comment:
a little typo in the function name I guess?  it should be get_pilot_desired_rpy_rate_cd probably?

Reply:
no, r and p are attitudes, y is a yaw rate
what name do you suggest?